k8s/base/04_ingress_routes.yaml

```diff
@@ -7,12 +7,12 @@ spec:
   entryPoints:
     - web
   routes:
-  - match: Host(`localhost`) && PathPrefix(`/api`)
+  - match: Host(`local.votifica.com`) && PathPrefix(`/api`)
```

Comment:
It's useful because backend won't allow connections from `localhost` due to ALLOWED_HOSTS settings. When testing locally, it's enough to add this `local.votifica.com` to `/etc/hosts` to test the deployment locally.